docs/INSTALL-email.md

```diff
@@ -17,7 +17,7 @@ Enter these values when prompted by `./discourse-setup` per the [install guide](
     SMTP password?       [SMTP credentials for your domain under [SMTP-Key tab](https://app.brevo.com/settings/keys/smtp)]
     Port:                587
     
-#### [Mailgun][gun] &mdash; 5k emails/month on a 3 month trial
+#### [Mailgun][gun] &mdash; 50k emails/month on a 30 day trial OR 10k emails/month with no trial
```

Comment:
<img width="1355" alt="Screenshot 2025-04-09 at 9 54 49 AM" src="https://github.com/user-attachments/assets/19e19d05-0aeb-4578-a290-085bf61b66a7" />

<img width="399" alt="Screenshot 2025-04-09 at 9 55 24 AM" src="https://github.com/user-attachments/assets/38170f5d-8388-4372-9d92-858f0f7fcb09" />


Can confirm this is what I see on the website as well.